packages/runtime/test-runtime-utils/src/validateAssertionError.ts

```diff
@@ -27,7 +27,8 @@ export function validateAssertionError(
 	error: Error,
 	expectedErrorMsg: string | RegExp,
 ): boolean {
-	const mappedMsg = (shortCodeMap[error.message] as string) ?? error.message;
+	const errorMessageNoPrefix = error.message.replace(/^FF Assert: /, "");
```

Comment:
can we just update the map the include the whole message, rather than needing a processing step.